src/tests/JIT/Regression/JitBlue/Runtime_98068/Runtime_98068.cs

```diff
@@ -0,0 +1,754 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+public static class Runtime_98068
+{
+    [Fact]
+    [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.AggressiveOptimization)]
```

Comment:
I think most pipelines disable TC anyway so there is no need in AggressiveOptimization. At the same time it'd be nice to make sure that Tier0 (hence, the non-intrinsic impl) passes these as well